backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3725,4 +3725,29 @@ databaseChangeLog:
             GRANT SELECT ON ${database.defaultSchemaName}.result TO ${noPhiUsername};
       rollback:
         sql: |
-          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
\ No newline at end of file
+          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
+  - changeSet:
+      id: add-foreign-key-indices-to-result-table
+      author: nclyde@skylight.digital
+      comment: Add indices to result table to speed up queries
+      changes:
+        - tagDatabase:
+          tag: add-foreign-key-indices-to-result-table
```

Comment:
Tag is not being read correctly, I am assuming its not in the correct yaml place, btw also noticed the last migration had the same issue

Reply:
Fixed!